.gitignore

```diff
@@ -6,3 +6,4 @@ coverage/*
 .htaccess
 dist/*
 version
+.vs/
```

Comment:
That doesn't look like a ghost graphic, in future please use appropriate commits for each change.
Also, why just `/`? other ignores are either `/*` or nothing